docs/modules/ROOT/pages/tutorials/tutorial.adoc

```diff
@@ -0,0 +1,378 @@
+* `kubectl get pods` to find the name of the WordPress pod.
+* `kubectl exec wordpress-567cd5ddbb-kjnt5 -it -- bash`
+
+TIP: In K9s you can get a shell into a pod just by selecting it and hitting the kbd:[S] key!
+
+[start=2]
+. Replace the files inside the pod with the ones from the restore:
+
+....
+$ cd /var/www/html # or whatever value has `mountPath`
+$ mv data /
+$ rm -r *
+$ cp -R /data/wordpress-pvc/* .
+....
+
+[start=3]
+. Since it is a PHP application, nothing else is needed for this pod.
+
+In the case of the MariaDB pod, the application must be restarted after restoring the files:
+
+[start=4]
+. Follow the same logic as previously:
+
+....
+$ cd /var/lib/mysql # Attention! Different path!
+$ mv data /
+$ rm -r *
+$ cp -R /data/mariadb-pvc/* .
+....
+
+[start=5]
+. Delete the MariaDB pod, and a new one will be immediately created according to the deployment: `kubectl delete pod mariadb-56cd59d499-jrkr7`
+
+Now refresh your WordPress browser window and you should see the previous state of the WordPress installation restored, working and looking as expected!
+
+.WordPress website restored
+image::tutorial/wordpress-restored.png[]
+
+[[step_6]]
+=== Cleaning up the backup pods
+
+NOTE: The operations of this step can be executed at once using the `scripts/6_delete_restore_pods.sh` script.
+
+Whenever K8up performs a backup, it creates a job pod; the one we created previously can be manually deleted using the command:
+
+`kubectl delete -f k8up/restore-wordpress.yaml`
+
+[[step_7]]
+=== Scheduling regular backups
+
+NOTE: The operations of this step can be executed at once using the `scripts/7_schedule.sh` script.
+
+Instead of performing backups manually, you can also set a schedule for backups. This requires specifying the schedule in `cron` format.
+
+[source,yaml]
+....
+backup:
+  schedule: '*/2 * * * *'    # backup every 2 minutes
+  keepJobs: 4
+  promURL: http://minio:9000
+....
+
+TIP: Use https://crontab.guru/[crontab.guru] to help you set up complex schedule formats in `cron` syntax.
+
+The schedule can also specify `archive` and `check` tasks to be executed regularly.
+
+[source,yaml]
+....
+archive:
+  schedule: '0 0 1 * *'       # archive every week
+  restoreMethod:
+    s3:
+      endpoint: http://minio:9000
+      bucket: archive
+      accessKeyIDSecretRef:
+        name: backup-credentials
+        key: username
+      secretAccessKeySecretRef:
+        name: backup-credentials
+        key: password
+check:
+  schedule: '0 1 * * 1'      # monthly check
+  promURL: http://minio:9000
+....
+
+Run the `kubectl apply -f k8up/schedule.yaml` command. This will setup an automatic schedule to backup the PVCs every 5 minutes (for minutes that are divisors of 5).
+
+Wait for at most 2 minutes, and run the `restic snapshots` to see more backups piling up in the repository.
+
+TIP: Running the `watch restic snapshots` command will give you a live console with your current snapshots on a terminal window, updated every 2 seconds.
```

Comment:
```suggestion
TIP: Running the `watch restic snapshots` command will rerun restic every 2 seconds and update the output so that you don't have to do it yourself.
```